diagnostic.md

```diff
@@ -27,7 +27,13 @@ on the body element. Use that variable to increase font-size of `h1`
 elements by 30%.
 
 ```scss
-// your response here
+$font-size =
+
+$quote-font-size: (font-size * .30 + font-size);
+
+h1 {
+  font-size: $base-font-size;
+}
```

Comment:
Not quite. At the moment, you're setting the h1 font size to `$base-font-size`, when you should be setting it to a size that's 30% larger.